### PR TITLE
Some minor cleanups

### PR DESCRIPTION
**Pull request by oscargus**
A little bit of everything... Maybe one should wait until after 3.3 is released before merging this?

### CODE REVIEW

File: src/main/java/net/sf/jabref/model/entry/Author.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public static String addDotIfAbbreviation(String name) {
 
             if(currentChar == '.') {
                 // A.A. -> A. A.
-                if(i + 1 < name.length() && Character.isUpperCase(name.charAt(i + 1))) {
+                if(((i + 1) < name.length()) && Character.isUpperCase(name.charAt(i + 1))) {
```
Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/model/entry/Author.java]: I find it more readable without the brackets...

**Comment by tobiasdiez** [comment]
LGTM :+1:

File: src/main/java/net/sf/jabref/model/entry/Author.java (hunk 1 above)
Comment (koppor) [inline src/main/java/net/sf/jabref/model/entry/Author.java]: Think, this is the eclipse setting we agreed about half year ago. We add brackets as much as possible to help readers not familiar with operator precedence to understand the code.

Comment (oscargus) [inline src/main/java/net/sf/jabref/model/entry/Author.java]: That is an automatic Eclipse addition... All brackets or outer brackets? Outer brackets I find quite useful, but I agree that the ones surrounding i + 1 could be skipped. There are even some more annoying cases of these brackets, e.g. (or a quite similar case)

```
j = k - l + m;
```
becomes
```
j = (k - l) + m;
```

Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/model/entry/Author.java]: Oh ok, you already had a discussion about it, never mind.

**Pull request merged**